.github/workflows/scheduled.yaml

```diff
@@ -22,3 +23,19 @@ jobs:
 
       - name: Run
         run: cargo test --test wasm-generation --test standard
+
+  notification:
+    name: Notification
+    runs-on: ubuntu-latest
+    needs: property-testing
+    if: ${{ failure() }}
+    steps:
+    - name: Notify on Failure
+      uses: maxkomarychev/oction-create-issue@v0.7.1
+      with:
+        token: ${{ secrets.GH_TOKEN }}
+        title: "Property testing coverage has failed"
+        body: "FYI & A: @stacks-network/clarity-wasm"
```

Comment:
Can we link to the failure somehow? This action looks useful: https://github.com/marketplace/actions/failed-build-issue.

Reply:
Good idea, Brice! Changed to this action and now we have a link to the workflow that has failed.